pkg/vpcmodel/abstractVPC.go

```diff
@@ -21,6 +21,7 @@ const (
 type VPCResourceIntf interface {
 	UID() string
 	Name() string
+	NameForAnalyzerOut() string
```

Comment:
still confusing/unclear functionality for `Name` / `NameForAnalyzerOut` / `ExtendedName` / `ExtendedPrefix`.

- Does the interface need to have all these methods?
- what is the use case for each method?
- which methods are overriden for certain types? is this required? (cannot be implemented explicitly instead of changing expected behavior of certain method?) 
- is there a method that always returns the original resource name from the config object?

Reply:
> * is there a method that always returns the original resource name from the config object?

Name()
consider renaming to resourceName() + doc

> Does the interface need to have all these methods?

we need Name() and NameForAnalyzerOut , ExtendedName and  ExtendedPrefix will check if we can merge it with NameForAnalyzerOut